app/validators/contact_message_validator.rb

```diff
@@ -0,0 +1,32 @@
+module ContactMessageValidator
+  def self.validate(message)
+    message.present? &&
+      validate_name(message) &&
+      validate_email(message)
+  end
+
+  private
+
+  def self.validate_name(message)
+    if !message.name.present?
+      message.errors.add(:name, :blank)
+      false
+    else
+      true
+    end
+  end
+
+  def self.validate_email(message)
```

Comment:
`private` (on line 8) does not make singleton methods private. Use `private_class_method` or `private` inside a `class << self` block instead.